wit/couchbase/sqlpp.wit

```diff
@@ -0,0 +1,11 @@
+package wasmcloud:couchbase@0.1.0-draft;
+
+/// Supports arbitrary queries written in SQL++ (formerly known as N1QL) on a Couchbase cluster.
+///
+/// Reference: https://docs.couchbase.com/server/current/n1ql/n1ql-language-reference/index.html
+interface sqlpp {
+  use sqlpp-types.{sqlpp-value, sqlpp-query-error};
+
+  /// Perform a N1QL query 
+  query: func(query: string, params: list<sqlpp-value>) -> result<sqlpp-value, sqlpp-query-error>;
```

Comment:
FTS accepts options, is there a reason for not accepting `QueryOptions` here ?

Reply:
I think you meant SQL++ here rather than FTS, and it's certainly missing, will add!

- [ ] Add missing options for sqlpp